msgid ""
msgstr ""
"Project-Id-Version: pylav\n"
"POT-Creation-Date: 2023-01-31 23:27+0000\n"
"PO-Revision-Date: 2023-01-31 23:42\n"
"Last-Translator: \n"
"Language-Team: French\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: redgettext 3.4.2\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"
"X-Crowdin-Project: pylav\n"
"X-Crowdin-Project-ID: 535244\n"
"X-Crowdin-Language: fr\n"
"X-Crowdin-File: /lib/pylav/extension/bundled_node/locales/messages.pot\n"
"X-Crowdin-File-ID: 3385\n"
"Language: fr_FR\n"

#: pylav/extension/bundled_node/manager.py:227
msgid "You are attempting to run the managed Lavalink node on an unsupported machine architecture"
msgstr "Vous tentez d’exécuter le nœud Lavalink géré sur une architecture de machine non prise en charge"

#: pylav/extension/bundled_node/manager.py:452
msgid "The output of `{java_exc} -version` was unexpected\n"
"{version_info}"
msgstr "La sortie de '{java_exc} -version' était inattendu\n"
"{version_info}"

#: pylav/extension/bundled_node/manager.py:472
msgid "Port {port} already in use. Managed Lavalink startup aborted"
msgstr "Port {port} déjà utilisé. Démarrage géré de Lavalink avorté"

#: pylav/extension/bundled_node/manager.py:477
msgid "Lavalink failed to start: {line}"
msgstr "Lavalink n’a pas pu démarrer : {line}"

